back-end/routes/stitchRoutes.js

```diff
@@ -117,6 +151,44 @@ router.patch('/title', async (req, res) => {
     }
 });
 
+router.patch('/preferences', async (req, res) => {
+    const { stitchId, mood, dance, mix, explore } = req.body;
+
+    const totalPreferences = mood + dance + mix;
+
+    let moodWeight = null;
+    let danceWeight = null;
+    let mixWeight = null;
```

Comment:
I think we'd rather initialize with 0 here, and just do `if (totalPreferences !== 0) { ... }` below to save an `else` branch.